packages/components/src/utils/card-style.ts

```diff
@@ -0,0 +1,51 @@
+import { Card } from '@models/Card';
+import { isPlayingCard } from '@models/PlayingCard';
+import { isUnoCard } from '@models/UnoCard';
+import { PlayingCardDimensions } from '@atoms/SvgPlayingCard';
+import { UnoCardDimensions } from '@atoms/UnoCardFace';
+import { Dimensions } from '@models/Dimensions';
+import { calculateCardDimensions } from './card-dimensions';
+
+export interface CardTypeStyle {
+  /** Default card height - used to calculate card ratio */
+  dimensions: Dimensions;
+  /** Card border radius */
+  borderRadius?: number;
+}
+
+export type CustomCardSytleFn = (card: Card, currentStyle?: CardTypeStyle) => CardTypeStyle;
```

Comment:
typo: `CustomCardSytleFn` => `CustomCardStyleFn`
Also, I would not use abbreviation in that case => `CustomCardStyleFunction`

Reply:
I changed to Factory